lib/tether/tether.go

```diff
@@ -42,11 +44,16 @@ import (
 )
 
 const (
-	// The maximum number of records to keep for restarting processes
+	// MaxDeathRecords The maximum number of records to keep for restarting processes
 	MaxDeathRecords = 5
 
 	// the length of a truncated ID for use as hostname
 	shortLen = 12
+
+	// temp directory to copy existing data to mounts
+	bindDir = "/.bind"
+	// markfile to avoid re-copying existing data to mounts
+	mountsCopied = "/.mountscopied"
```

Comment:
I think we should not copy multiple times. 

As per docker's implementation, this operation is performed only once when the container is created. 

Also it looks like docker copies *ALL* the data and not the new and non-existent ones.   

https://github.com/vmware/vic/blob/master/vendor/github.com/docker/docker/container/container_unix.go#L379-L397